### ISSUE TITLE
DPL-306 - ONT - Add library type support

### ISSUE THREAD

**Issue by stevieing** [sanger/traction-service#726]
**User story**
ONT - Add library type support. We mentioned before that if a new PacBio workflow comes on line, IsoSeq for example, it can be bolted on to the LIMS as a library type. It would be good to have a similar set up for ONT so that if a new workflow comes on line we can get it added easily if the requirement is high enough.

**Who are the primary contacts for this story**
Steve / James / Alex

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] add a list of acceptance criteria here
- [ ] ...

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

References
This story has a non-blocking relationship with:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by stevieing** [comment]
linked to sanger/traction-ui#720

**Comment by stevieing** [comment]
Already done.
